### PR TITLE
civetweb: skip broken Conan v1.51.0 version

### PR DESCRIPTION
**Pull request by dornbirndevelops**
Specify library name and version:  **civetweb/all**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

A different implementation of conan.tools.scm.Version in v1.51.0 causes the recipe to fail.
The implementation has been fixed in v1.51.2.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by dornbirndevelops** [comment]
@uilianries could you please update the Conan client version of the CI agent to a newer version?

File: recipes/civetweb/all/conanfile.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
 import os
 
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.51.2"
```
Comment (SpaceIm) [inline recipes/civetweb/all/conanfile.py:8]:
```suggestion
required_conan_version = ">=1.50.2 <1.51.0 || >=1.51.2"
```

**Review by SpaceIm — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Comment by uilianries** [comment]
@dornbirndevelops it's not possible. We used to follow the latest version, but we had problems with new Conan bugs which affected all CCI, so we decided to use one previous minor version.

**Comment by dornbirndevelops** [comment]
> @dornbirndevelops it's not possible. We used to follow the latest version, but we had problems with new Conan bugs which affected all CCI, so we decided to use one previous minor version.

it's okay, @SpaceIm had a very good idea for the proposed changes. the version used by the agents should be fine for now :)

**Pull request merged**